lib/inferno/dsl/runnable.rb

```diff
@@ -213,6 +213,17 @@ def title(new_title = nil)
         @title = new_title
       end
 
+      # Set/Get a runnable's short title
+      #
+      # @param new_short_title [String]
+      # @return [String] short title, or title if no short title defined
+      def short_title(new_short_title = nil)
+        return @title if new_short_title.nil? && @short_title.nil?
```

Comment:
To me, this seems like a display concern that should be handled in the front end rather than the back end.

Reply:
As I was writing out the PR I started having the same thoughts.  I think this is especially true because of the unexpected difference in behavior between description/short_description and title/short_title.  Pushed an update.